mlir/test/Dialect/Vector/vector-transfer-permutation-lowering.mlir

```diff
@@ -1,14 +1,81 @@
 // RUN: mlir-opt %s --transform-interpreter --split-input-file | FileCheck %s
 
 ///----------------------------------------------------------------------------------------
-/// vector.transfer_write
+/// vector.transfer_write -> vector.transpose + vector.transfer_read
 ///----------------------------------------------------------------------------------------
-/// Input: 
-///   * vector.transfer_write op with a map which _is not_ the permutation of a
-///     minor identity
+/// Input:
+///   * vector.transfer_write op with a permutation that under a transpose
+///     _would be_ a permutation of a minor identity
 /// Output:
-///   * vector.broadcast + vector.transfer_write with a map which _is_ the permutation of a
+///   * vector.transpose + vector.transfer_write with a map which _is_ a
+///     permutation of a minor identity
```

Comment:
It is not a permutation of a minor identity but a minor identity per se.
See examples in 
https://github.com/llvm/llvm-project/blob/77db8b08c8b186c2625f8dfb26bb976561b43c4c/mlir/lib/Dialect/Vector/Transforms/LowerVectorTransfer.cpp#L156-L174
outputs are all of identity like.  The goal of those patterns is to generate identity permutation map.

Reply:
That's a very good point, thanks!

I read this comment a while back and just have it hard-coded in my head that it's "the permutation of a minor identiy":
https://github.com/llvm/llvm-project/blob/77db8b08c8b186c2625f8dfb26bb976561b43c4c/mlir/lib/Dialect/Vector/Transforms/LowerVectorTransfer.cpp#L231-L245